doc/api/worker_threads.md

```diff
@@ -360,11 +360,12 @@ Notable differences inside a Worker environment are:
   being invoked.
 - IPC channels from parent processes are not accessible.
 - The [`trace_events`][] module is not supported.
+- Native add-ons can only be loaded from multiple threads if they fulfill
+  [certain conditions][Addons worker support].
 
 Currently, the following differences also exist until they are addressed:
```

Comment:
Should this remain as the list now or be replaced with a note about inspector support?

Reply:
Oh, right, the `inspector` module is also available by now :)

I think it should just be removed.